### ISSUE TITLE
fix: StdinException when using enums on Windows (10/11)

### ISSUE THREAD

**Issue by LukeMoody01** [felangel/mason#397]
**Description**

Using enums in your brick.yaml will cause a StdinException when executing `mason make <brick>`

**Steps To Reproduce**

1. Create a new brick
2. Have an enum variable:
```
  color:
    type: enum
    default: red
    description: The color of the sky
    prompt: What is the color of the sky?
    values:
      - red
      - yellow
      - blue
```
4. Run the `mason make <brick>`  *On Windows (10/11)
5. See error

**Expected Behavior**

The developer should be able to use enum values on Window devices 

**Screenshots**
When you get to the enum variable
![image](https://user-images.githubusercontent.com/73887660/172610337-5fc22417-1509-4e37-8f95-c7b85a7e4b06.png)

**Additional Context**

Edit: Checked on Mac, and everything is working as expected. This issue seems to only be on Windows (10/11)

**Comment by felangel** [comment]
Hi @LukeMoody01 👋
Thanks for opening an issue!

I’ll take a closer look shortly, sorry for the inconvenience!

**Comment by LukeMoody01** [comment]
> Hi @LukeMoody01 👋 Thanks for opening an issue!
> 
> I’ll take a closer look shortly, sorry for the inconvenience!

It's no worries! I am sticking to string value types for now, so it is not too much of an inconvenience!

Thanks for the quick response!

**Comment by felangel** [comment]
This should be resolved in mason_logger v0.1.0-dev.12 🎉

**Comment by LukeMoody01** [comment]
Amazing! Thanks @felangel!
